### PR TITLE
add Homebrew instructions

### PR DESCRIPTION
**Pull request by btbytes**
## Purpose

Improve documentation

## Related PRs

Only the submitted PR

## Special notes

- only documentation change. No tests were created/modified.

### CODE REVIEW

**Comment by TylerJewell** [comment]
Hmm, I think this may not be the best place for this type of thing. I think it's because we should create a special section on the installation / download page for this information, and then avoid repeating this info on the Getting Started page.  We are likely to have many more of these installation packages, and so it would get to be too noisy on the Getting Started.

**Comment by btbytes** [comment]
Considering ballerina is a developer tool and homebrew is the most popular way for developers to install software on a mac (AFAIK), `brew install ballerina` should perhaps take precedence over binary download + install.

**Comment by TylerJewell** [comment]
We had numerous back-and-forth about that very topic when we did our GTM planning offsite at the end of March.  

We opted against making the package managers the default installation because we cannot guarantee (with many of them) that they get propagation to the proper version at the time of a release. So we are not able to put them through all of the verification processes that ensures quality and many package managers have different propagation of versions over time, along with some approval processes. 

So the only guaranteed consistent way for all of the OS is the approach that we have - and then we wanted to iteratively evolve the downloads page with all of the package manager options available. It's a bit similar to how Rust has handled it, which is pretty elegant.

**Comment by TylerJewell** [comment]
BTW - thanks for getting involved and making the recommendation. This PR is just triggering the resurgence of all the discussions that were had around package managers, and need to figure out whether now is a good time to ask R&D core team to revisit how they add all of the package managers as part of our CI system, so that they magically show up everywhere they need to be.

File: website/two-column-pages/docs/learn/getting-started.md [hunk 1]
```diff
@@ -30,6 +30,12 @@ If you are upgrading from an older version of Ballerina, you can simply download
 
 The package automatically sets your PATH environment variable for you. You may need to restart any open Terminal sessions for the change to take effect.
 
+Alternatively, if you have [Homebrew](https://brew.sh/) installed on your Mac, you can install the ballerina package with:
```
Comment (samgnaniah) [inline website/two-column-pages/docs/learn/getting-started.md]: Thank you for this addition. Can you change 'ballerina' to 'Ballerina'? Apart from that this looks good.

Comment (btbytes) [reply website/two-column-pages/docs/learn/getting-started.md]: done.

**Comment by samgnaniah** [comment]
Tested this. It doesn't work for Ballerina 0.975.0 as yet.

**Comment by rasika** [comment]
We can merge this now. `brew install ballerina` will install the latest version(aka. v0.975.1). And for the the future versions it is just a matter of sending a PR to homebrew as https://github.com/Homebrew/homebrew-core/pull/29696

**Comment by samgnaniah** [comment]
I'm closing this as it's been open for a bit too long. We will address this once we have the strategy for package managers figured out.

**Pull request closed without merging**